prisma/schema.prisma

```diff
@@ -93,17 +96,36 @@ model Story {
 model StoryContribution {
   id String @id @default(auto()) @map("_id") @db.ObjectId
 
-  user   User   @relation(fields: [userId], references: [id])
-  userId String @db.ObjectId
+  //user   User   @relation(fields: [userId], references: [id])
+  //userId String @db.ObjectId
+
+  contributor Contributor? @relation(fields: [contributorId], references: [id])
+  contributorId String? @db.ObjectId
 
   story   Story  @relation(fields: [storyId], references: [id], onDelete: NoAction)
   storyId String @db.ObjectId
 
   contributionType ContributionType
+  otherContributorType String?
+  otherContributorCredit String?
```

Comment:
are these to write-in one-off contribution types?

Reply:
I'm not too sure on what write-in one-off means

Reply:
sorry, are these for single-use contributor types? For example, if there was some category like "Designer" that is only used in one story or something?